akka-discovery-azure-api/src/main/resources/reference.conf

```
@@ -0,0 +1,60 @@
+######################################################
+# Akka Service Discovery Azure Config                  #
+######################################################
+
+akka.discovery {
+  # Set the following in your application.conf if you want to use this discovery mechanism:
+  # method = aws-api-ec2-tag-based
+  azure-rbac-aks-api {
+    class = akka.discovery.kubernetes.AzureRbacAksServiceDiscovery
+
+    authority-host = "https://login.microsoftonline.com/"
+    authority-host = ${?AZURE_AUTHORITY_HOST}
+
+    // can't be empty
```

mixing comment styles, stick with one
```suggestion
    # can't be empty
```

---

Yep, my bad.